app/src/main/java/com/salman/roomtask/ui/home/HomeFragment.kt

```diff
@@ -39,13 +49,53 @@ class HomeFragment : Fragment() {
         viewModel.categories.observe(viewLifecycleOwner) { categories ->
             binding.chipGroupCategories.submitList(categories) { category ->
                 // TODO navigate to details fragment with category id
+                findNavController().navigate(R.id.detailsFragment, Bundle().apply {
```

Comment:
I think using the nav args plugin would make this much easier, safer, and more readable